Makefile

```diff
@@ -62,6 +63,7 @@ docker-clean:
 clean:
 	rm -f composer.lock .php-cs-fixer.cache .phpunit.result.cache
 	rm -Rf vendor
+	cd tests/integration && $(DCO) down -v --remove-orphans --rmi local
```

Comment:
any reason to add `--rmi local`

Reply:
so that when we run `clean` any image that was build will be deleted in case something changed